tests/wallets2descriptors.rs

```diff
@@ -0,0 +1,84 @@
+#[cfg(feature = "wallet_file")]
+use bdk::{bitcoin::Network, database::MemoryDatabase, wallet::AddressIndex, Wallet};
+#[cfg(feature = "wallet_file")]
+use libelectrum2descriptors::electrum_wallet_file::{electrum_wallet_to_descriptors, get_json_from_wallet_file};
+#[cfg(feature = "wallet_file")]
+use rstest::rstest;
+#[cfg(feature = "wallet_file")]
+use std::path::{Path, PathBuf};
+
+#[cfg(feature = "wallet_file")]
+#[rstest]
+#[case("default_legacy", 
```

Comment:
I see rstest support more friendly test naming in report by using this syntax:
`#[case::default_legacy("default_legacy",`

will result in:
`test test_wallet::case_1_default_legacy ... ok`

instead of just:
`test test_wallet::case_1 ... ok`

Reply:
Cool